src/view/Router.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import SmartQuestionsList from "./SmartQuestionsList";
+import SmartCreateQuestion from "./SmartCreateQuestion";
+import SmartQuestionDetails from "./SmartQuestionDetails";
+
+const Router = ({route}) => (
```

Comment:
As you are using the react router now, you can safely remove this and the smart router.